### PR TITLE
Submit gentx for Blockdaemon

### PR DESCRIPTION
**Pull request by JackPMallon**
## Overview

Submitting gentx for Blockdaemon

### CODE REVIEW

**Comment by YazzyYaz** [comment]
hey @JackPMallon you're adding whitespace, hence why it's failing

**Review by liamsi — APPROVED** [review]
jq does not block merging

File: celestia/Blockdaemon.txt [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+        "delegator_address": "celestia1evu846d408jdz5ju4883gtq9pnjvs8hzq8z2c9",
+        "validator_address": "celestiavaloper1evu846d408jdz5ju4883gtq9pnjvs8hz9cqnwr",
+        "pubkey": {
+          "@type": "/cosmos.crypto.ed25519.PubKey",
+          "key": "xwbuyz6bUNd+Ef79w0ysCU4qgDru1OZTn/yHnPaw5JE="
+        },
+        "value": {
+          "denom": "utia",
+          "amount": "66400000000"
+        }
```
Comment (evan-forbes) [inline celestia/Blockdaemon.txt:28]: I think this was added by accident @JackPMallon its not the end of the world though we can just delete this file

**Comment by YazzyYaz** [comment]
can you delete the Blockdaemon.txt file unless you change it to a peer id. only Blockdaemon.json should be used for submitting the gentx

**Comment by JackPMallon** [comment]
Okay my bad I thought they had to all pass. Everything should be correct now. Let me know if you guys need anything else.

**Review by adlerjohn — APPROVED** [review]

**Pull request merged**